### PR TITLE
Tpcc benchmark log

### PR DESCRIPTION
**Pull request by thepinetree**
Addresses Issue #686

### CODE REVIEW

**Comment by apavlo** [comment]
@thepinetree I will finish this PR.

File: benchmark/integration/tpcc_benchmark.cpp [hunk 1]
```diff
@@ -59,6 +56,8 @@ class TPCCBenchmark : public benchmark::Fixture {
   storage::GarbageCollectorThread *gc_thread_ = nullptr;
   const std::chrono::milliseconds gc_period_{10};
   const std::chrono::milliseconds metrics_period_{100};
+
+  const char *logfile_ = (std::getenv("LOGFILE_NAME") == nullptr ? "/tmp/tpcc.log" : std::getenv("LOGFILE_NAME"));
```
Comment (apavlo) [inline benchmark/integration/tpcc_benchmark.cpp]: I will fix. I think I have to start a review first to push into your branch?

Comment (thepinetree) [reply benchmark/integration/tpcc_benchmark.cpp]: The changes on your branch look good to me... do I have to do anything for you to push into this branch?

Comment (apavlo) [reply benchmark/integration/tpcc_benchmark.cpp]: No I got it.

**Comment by apavlo** [comment]
Spurious benchmark regressions. I will merge.

**Pull request merged**